src/gazebo_worlds/package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>gazebo_worlds</name>
```

Comment:
There already is [gazebo_worlds](http://wiki.ros.org/gazebo_worlds) package in ROS. Consider using robot prefix
```suggestion
  <name>sirius_worlds</name>
```